core/src/validator.rs

```diff
@@ -517,8 +519,9 @@ impl ValidatorTpuConfig {
             ..Default::default()
         };
 
-        // vote and tpu_fwd share the same characteristics -- disallow non-staked connections:
+        // vote/tpu_fwd/alpenglow share the same characteristics -- disallow non-staked connections:
         let vote_quic_server_config = tpu_fwd_quic_server_config.clone();
+        let alpenglow_quic_server_config = tpu_fwd_quic_server_config.clone();
```

Comment:
This is probably okay, but I would like us to review all configurations and make sure it makes sense for alpenglow. Do they really match the current TPU forward configs other than disallowing non-staked connections?